### PR TITLE
Fix IL bug: delete job of lc when deploy completed

### PR DESCRIPTION
**Pull request by JimmyYang20**

### CODE REVIEW

File: pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go [hunk 1]
```diff
@@ -450,10 +450,16 @@ func (im *Manager) Insert(message *clienttypes.Message) error {
...
-		if err := os.RemoveAll(dir); err != nil {
-			return fmt.Errorf("failed to delete the dir(%s): %w", dir, err)
+		localHostDir := util.AddPrefixPath(im.VolumeMountPrefix, filepath.Dir(configFile))
+		dir, err := ioutil.ReadDir(localHostDir)
+		if err != nil {
+			return err
+		}
+		for _, d := range dir {
+			file := filepath.Join([]string{localHostDir, d.Name()}...)
+			if e := os.RemoveAll(file); err != nil {
```
Comment (llhuii) [inline pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go:460]:
```suggestion
			if e := os.RemoveAll(file); e != nil {
```

File: pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go [hunk 2]
```diff
@@ -450,10 +450,16 @@ func (im *Manager) Insert(message *clienttypes.Message) error {
 // deleteModelHotUpdateData deletes the local data of model hot update
 func (im *Manager) deleteModelHotUpdateData(job *Job) error {
 	if configFile, ok := job.ObjectMeta.Annotations[runtime.ModelHotUpdateAnnotationsKey]; ok {
-		localHostDir := filepath.Dir(configFile)
-		dir := util.AddPrefixPath(im.VolumeMountPrefix, localHostDir)
-		if err := os.RemoveAll(dir); err != nil {
-			return fmt.Errorf("failed to delete the dir(%s): %w", dir, err)
+		localHostDir := util.AddPrefixPath(im.VolumeMountPrefix, filepath.Dir(configFile))
+		dir, err := ioutil.ReadDir(localHostDir)
```
Comment (llhuii) [inline pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go:454]: When has localHostDir been created?

Comment (llhuii) [inline pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go:454]:
```suggestion
		files, err := ioutil.ReadDir(localHostDir)
```

File: pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go (hunk 1 above)
Comment (JimmyYang20) [reply pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go:460]: fix it

File: pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go (hunk 2 above)
Comment (JimmyYang20) [reply pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go:454]: when start infer worker

Comment (JimmyYang20) [reply pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go]: ok ,fix it

**Pull request closed without merging**